.github/workflows/script-evaluation-test.yml

```diff
@@ -0,0 +1,37 @@
+# A nightly job which downloads script evaluation dumps from S3 and runs a regression test.
+name: Script Evaluation Test
+on:
+  schedule:
+    - cron: 30 3 * * * # 3:30am every day 
+
+jobs: 
+  script-evaluation-test: 
+    runs-on: [self-hosted, plutus-benchmark]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.3.0
+
+      - name: Download and Unzip Dump Files
+        run: |
+          export LOCAL_DIR="$HOME/mainnet-script-dump-downloaded" 
+          nix develop --no-warn-dirty --accept-flake-config --command \
+            bash ./scripts/s3-sync-unzip.sh s3://plutus/mainnet-script-dump/ \*.event.bz2
```

Comment:
Since we're running on a persistent machine, is there a way we can save this somewhere persistent and only sync the remainder?

Reply:
I think the file `$HOME/mainnet-script-dump-downloaded` is going to persist across runs, no?

Reply:
Oh I see. I also don't know if that's true. Maybe we should check with @Pacman99 .

I also don't know if that script does things incrementally, probably a question for @zliu41